src/interface/src/app/shared/nav-bar/nav-bar.component.spec.ts

```diff
@@ -12,7 +11,6 @@ describe('NavBarComponent', () => {
   beforeEach(async () => {
     await TestBed.configureTestingModule({
       declarations: [NavBarComponent],
-      schemas: [CUSTOM_ELEMENTS_SCHEMA],
```

Comment:
FYI - I can't recall what will happen here, but it was added very recently as part of nav bar work that was being done (which requires a flag that you may not have enabled).  Is there anything unwanted if you leave CUSTOM_ELEMENTS_SCHEMA in?

Reply:
Git erroneously marked part of this code as a merge conflict so I believe I resolved it incorrectly. It should be good to go now.